frontend/src/locales/de.json

```diff
@@ -498,6 +498,12 @@
     "datetime": {
       "vuetifyTimePickerFormat": "24hr"
     },
+    "info": {
+      "offline": {
+        "description": "Es werden keine Daten gespeichert.",
```

Comment:
That sounds a little harsh and could be misunderstood. How about something like "Keine Verbindung möglich. Aktuell können keine Änderungen gespeichert werden."

Reply:
I've tried to improve it.